setup.py

```diff
@@ -19,23 +19,16 @@ def requirements():
     """
     requirements_list = []
     for requirement in open("requirements.txt").read().rsplit("\n"):
-        if "python_version" in requirement:
-            if "> '3'" in requirement and int(sys.version_info[0]) == 3:
-                try:
+        if requirement != "":
+            try:
+                if '>' in requirement:
```

Comment:
i'm not sure if this works with every package.
for eg : tensorflow-gpu==1.15 is not imported as tensorflow-gpu but tensorflow

Reply:
i think it will work. I will commit the if block statement above try block, because currently I'm testing the setup.py file.
![setup](https://user-images.githubusercontent.com/45870070/121999654-30759700-cdcb-11eb-870e-6e0ddf54b612.png)